ApiUsage/ArmBased/Program.cs

```diff
@@ -280,5 +390,13 @@ public class Video
             [JsonPropertyName("state")]
             public string State { get; set; }
         }
+
+        public static void VerifyStatus(HttpResponseMessage response, System.Net.HttpStatusCode excpectedStatusCode = System.Net.HttpStatusCode.OK)
```

Comment:
Nice , I suggest not using default Value , so that each statement above will have C:LEAR view on what is the required result status should be without arriving to this method